internal/flag/flags.go

```diff
@@ -39,6 +39,7 @@ const (
 	Password                        = "password"                        // Password flag
 	X509Type                        = "x509Type"                        // X509Type flag
 	AWSIAMType                      = "awsIAMType"                      // AWSIAMType flag
+	DeleteAfterDate                 = "deleteAfterDate"                 // DeleteAfterDate flag
```

Comment:
[nit] usually defining the type is redundant
```suggestion
	DeleteAfter                 = "DeleteAfter"                 // DeleteAfter flag
```
Also `DeleteAfter` already exists in this file so we can just delete this line